ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -239,10 +425,83 @@ def create_sessions(
 
         Raises:
             ValueError: If the instrument type ID is empty or no reserved
-                sessions match the instrument type ID.
+                sessions matched the instrument type ID.
         """
         return self._create_sessions_core(session_constructor, instrument_type_id)
 
+    @requires_feature(SESSION_MANAGEMENT_2024Q1)
+    def get_connection(
+        self,
+        session_type: Type[TSession],
+        pin_or_relay_names: Union[str, Iterable[str], None] = None,
+        sites: Union[int, Iterable[int], None] = None,
```

Comment:
The single-connection version might as well take a single pin/relay and a single site, since there can only be one result.

Reply:
That is consistent with the LV API.